hal-core/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "hal-core"
 version = "0.1.0"
-authors = ["Eliza Weisman <eliza@elizas.website>"]
```

Comment:
hey

Reply:
whoop's...we should probably add an ixi too